docs/examples/academy/28. Aggregating Cargo Movements Data.ipynb

```diff
@@ -314,23 +266,23 @@
       "text/plain": [
        "              quantity\n",
        "loading_week          \n",
-       "21            16203392\n",
-       "22            19765442\n",
-       "23            21530598\n",
-       "24            21794264\n",
-       "25            25614818\n",
-       "26            19651398\n",
-       "27            24615334\n",
-       "28             5623297"
+       "9              6177861\n",
+       "10            26019118\n",
+       "11            36710668\n",
+       "12            28675170\n",
+       "13            27427342\n",
+       "14            17971104\n",
+       "15            31144565\n",
+       "16            22209895"
       ]
      },
-     "execution_count": 9,
+     "execution_count": 11,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "weekly_quantity = cms.groupby('loading_week').sum()\n",
+    "weekly_quantity = cms.groupby('loading_week').sum(numeric_only=True)\n",
```

Comment:
This is the only meaningful change in this file - it fixes an issue where the `.sum` method was not working after a Pandas upgrade.